testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveSparkCompatibility.java

```diff
@@ -187,6 +187,162 @@ public void testReadSparkCreatedTable(String sparkTableFormat, String expectedTr
         onSpark().executeQuery("DROP TABLE " + sparkTableName);
     }
 
+    @Test(groups = {HIVE_SPARK, PROFILE_SPECIFIC_TESTS}, dataProvider = "testReadSparkCreatedTableDataProvider")
+    public void testReadTrinoCreatedTable(String sparkTableFormat, String expectedTrinoTableFormat)
```

Comment:
If this is Trino-created, the arguments should be

`expectedTrinoTableFormat` -> `trinoTableFormat`
`sparkTableFormat` -> expected sparkTableFormat`